### PR TITLE
Chat application

### PR DESCRIPTION
**Pull request by MohamedAmine-C**

### CODE REVIEW

File: app/Chat.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace English;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Chat extends Model
+{
+    protected $table = 'chat';
```
Comment (CaddyDz) [inline app/Chat.php]: You don't need to explicitly tell what's the table in DB that the Model is representing as of Laravel 5.3

File: app/Http/Controllers/ChatController.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace English\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use English;
```
Comment (CaddyDz) [inline app/Http/Controllers/ChatController.php]: There's absolutely no need for this, use the Model directly
Change this to 
`use English\Chat;`

File: routes/web.php [hunk 3]
```diff
@@ -59,7 +59,7 @@
  */
 
 // Post statuses
-Route::middleware(['auth'])->group(function() {
+Route::middleware(['auth'])->group(function () {
```
Comment (CaddyDz) [inline routes/web.php]: This is violating the repository Code Style 😒

File: routes/web.php [hunk 4]
```diff
@@ -84,3 +84,11 @@
         'as' => 'status'
     ]);
 });
+/*
+ * Chat
+ */
+ Route::group(['prefix' => 'chat'], function () {
+     Route::get('send', 'ChatController@send');
+     Route::get('update', 'ChatController@update');
+     Route::get('{corespendent}', 'ChatController@show');
```
Comment (CaddyDz) [inline routes/web.php]: correspondent*

File: app/Http/Controllers/ChatController.php (hunk 2 above)
Comment (MohamedAmine-C) [inline app/Http/Controllers/ChatController.php]: without this line the application throw an error : Class 'English\Http\Controllers\English\Chat' not found
anyway it's better to use `use English\Chat;`

**Pull request merged**